index.bs

```diff
@@ -3754,6 +3781,39 @@ extension for transaction authorization.
     });
 </pre>
 
+## Aborting Authentication Operations ## {#sample-aborting}
+
+The below example showcases how a developer may use the AbortSignal parameter to abort an
+ongoing credential registration operation. Similiar procedure applies to an ongoing
+authentication operation. 
+
+<pre class="example" highlight="js">
+    const authAbortController = new AbortController(); 
+    const authAbortSignal = controller.signal;
+
+    var options = {
+        // A list  of options. 
+    }
+
+    navigator.credentials.create({"publicKey": options, authAbortSignal})
+        .then(function (attestation) {
+            // Register the user. 
+        }).catch(function (error) {
+            if (error = "AbortError") {
+                // Let the server know a key hasn't been created. 
```

Comment:
The exact error type probably doesn't affect what the server gets told. It might affect the UI shown to the user?